dtfs-central-api/src/repositories/utilisation-reports-repo/utilisation-report-sql.repo.ts

```diff
@@ -44,6 +44,8 @@ export const UtilisationReportRepo = SqlDbDataSource.getRepository(UtilisationRe
 
     if (options?.excludeNotReceived) {
       findByOptionsWhere.status = Not('REPORT_NOT_RECEIVED');
+      const notReceivedReports = await this.findBy(findByOptionsWhere);
+      return notReceivedReports.filter((report) => report.azureFileInfo);
```

Comment:
Because our join column exists on the `AzureFileInfo` table, we can't query against the `UtilisationReport` table for it. Therefore, we must either use a query builder (which would be quite complex in this case due to the difference combinations of `options`) or just filter once we get all reports from the `UtilisationReport` table. I've implemented the latter here

Reply:
Discussed with @ChristianMcc: the intended behaviour of this query was updated without my knowledge, and I didn't realise it was a change we wanted to prefer to query on status than the not uploaded flag. I'll revert this and the controllers which require a check for `azureFileInfo` being defined will be updated as well